cobigen-nest-server/package.json

```diff
@@ -1,64 +1,22 @@
 {
-  "name": "@devonfw/cobigen-nestserver",
-  "bin": "dist/main.js",
-  "version": "1.1.7",
-  "description": "Nestjs server to implement the input reader and merger for CobiGen",
-  "author": "CobiGen Team",
-  "license": "MIT",
+  "name": "cobigen-nest-server",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
-    "format": "prettier --write \"src/**/*.ts\"",
-    "start": "ts-node -r tsconfig-paths/register src/main.ts",
-    "start:dev": "nodemon",
-    "start:debug": "nodemon --config nodemon-debug.json",
-    "prestart:prod": "rimraf dist && npm run build",
-    "start:prod": "node dist/main.js",
-    "lint": "tslint -p tsconfig.json -c tslint.json",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "build": "npm run cleanup:root && npm run cleanup:packages && tsc -p tsconfig.build.json && npm run copy:rename && npm run compress",
+    "cleanup:root": "rimraf dist",
+    "cleanup:packages": "rimraf packages/*/dist/",
+    "copy:rename": "npm run copy && npm run rename",
+    "copy": "copyfiles -f \"node_modules/@devonfw/ts-merger/dist/ts-merger.umd.js\" \"node_modules/@devonfw/ts-merger/build/\"",
+    "rename": "node -e \"require('fs').rename('node_modules/@devonfw/ts-merger/build/ts-merger.umd.js', 'node_modules/@devonfw/ts-merger/build/index.js', function(err) { if (err) console.log(err); console.log('File successfully renamed!') })\"",
+    "compress": "terser node_modules/@devonfw/ts-merger/build/index.js  --compress --mangle --warn --output node_modules/@devonfw/ts-merger/build/index.js"
```

Comment:
you programmed all this stuff here by hand. Why didn't you use webpack? https://webpack.js.org/guides/getting-started/
Any reason?

Reply:
https://github.com/dan-kez/lerna-webpack-example